agent/tool-scripts/README.md

```diff
@@ -0,0 +1,219 @@
+# Tool Scripts (`agent/tool-scripts`)
+
+The "scripts" in this directory represent the "tools" that users can register
```

Comment:
Why are `scripts` and `tools` quoted?

Reply:
I removed the quotes on "scripts".  We quote "tools" because they are not the actual tools the run, but wrappers around the tools they represent, and are really part of the pbench-agent interface.